collections/go.mod

```diff
@@ -18,3 +18,5 @@ require (
 )
 
 replace cosmossdk.io/core/testing => ../core/testing
+
+replace cosmossdk.io/schema => ../schema
```

Comment:
Can we do this in a separate PR? I'm pretty sure this replace has a global effect on all go.mod's in the SDK. Instead if needed we can tag `schema` and then update collections.

Reply:
Sure let's have alpha.3. @aaronc is doing release for schema I think. I'll tag core once schema is tagged (to bump schema there).